File: specification/batch/resource-manager/Microsoft.Batch/stable/2023-05-01/examples-java/PoolUpdate_RemoveStartTask.java
import com.azure.resourcemanager.batch.models.Pool;
import com.azure.resourcemanager.batch.models.StartTask;

/** Samples for Pool Update. */
public final class Main {
    /*
     * x-ms-original-file: specification/batch/resource-manager/Microsoft.Batch/stable/2023-05-01/examples/PoolUpdate_RemoveStartTask.json
     */
    /**
     * Sample code: UpdatePool - Remove Start Task.
     *
     * @param manager Entry point to BatchManager.
     */
    public static void updatePoolRemoveStartTask(com.azure.resourcemanager.batch.BatchManager manager) {
        Pool resource =
            manager
                .pools()
                .getWithResponse(
                    "default-azurebatch-japaneast", "sampleacct", "testpool", com.azure.core.util.Context.NONE)
                .getValue();
        resource.update().withStartTask(new StartTask()).apply();
    }
}

File: specification/batch/resource-manager/Microsoft.Batch/stable/2022-10-01/examples-js/PoolUpdate_RemoveStartTask.js
const { BatchManagementClient } = require("@azure/arm-batch");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Updates the properties of an existing pool.
 *
 * @summary Updates the properties of an existing pool.
 * x-ms-original-file: specification/batch/resource-manager/Microsoft.Batch/stable/2022-10-01/examples/PoolUpdate_RemoveStartTask.json
 */
async function updatePoolRemoveStartTask() {
  const subscriptionId = "subid";
  const resourceGroupName = "default-azurebatch-japaneast";
  const accountName = "sampleacct";
  const poolName = "testpool";
  const parameters = { startTask: {} };
  const credential = new DefaultAzureCredential();
  const client = new BatchManagementClient(credential, subscriptionId);
  const result = await client.poolOperations.update(
    resourceGroupName,
    accountName,
    poolName,
    parameters
  );
  console.log(result);
}

updatePoolRemoveStartTask().catch(console.error);
